package.json

```diff
@@ -19,11 +19,11 @@
     "start": "webpack-dev-server --config webpack/dev.config.js",
     "start:light": "WEBPACK_BUILD_LIGHT=1 webpack-dev-server --config webpack/dev.config.js",
     "clean": "rm -rf dist",
-    "build-dev": "webpack --config webpack/es5.config.js --progress --display-error-details --color --mode=development",
-    "build": "webpack --config webpack/es5.config.js --progress --display-error-details --color",
+    "build-dev": "webpack --config webpack/es5.config.js --progress --color --mode=development",
+    "build": "webpack --config webpack/es5.config.js --progress --color",
     "postbuild-all": "bundlewatch",
-    "build-light-dev": "WEBPACK_BUILD_LIGHT=1 webpack --config webpack/es5.config.js --progress --display-error-details --color --mode=development",
```

Comment:
display-error-details is v4 deprecated, moved to the webpack config file : stats: 'errors-only'